File: Baekjoon/solved_ac/class2/2869.cpp
// 2869: 달팽이는 올라가고 싶다

#include <iostream>
using namespace std;

int main() {
  int A, B, V;
  cin >> A >> B >> V;

  int cnt = (V-A) / (A-B);
  if ((V-A) % (A-B) == 0) cnt++;
  else cnt += 2;

  cout << cnt;

  return 0;
}

File: Baekjoon/solved_ac/class2/2869.py
# 2869: 달팽이는 올라가고 싶다

A, B, V = list(map(int, input().split()))

cnt = (V - A)//(A - B)
if ((V - A) % (A - B) == 0): cnt += 1
else: cnt += 2

print (cnt)
